recipes/magic_enum/all/conanfile.py

```diff
@@ -39,7 +39,7 @@ def _compilers_minimum_version(self):
         }
 
     def layout(self):
-        basic_layout(self, src_folder="src")
+        cmake_layout(self)
```

Comment:
```suggestion
        cmake_layout(self, src_folder="src")
```

Reply:
Thank you, added change in [00457e3](https://github.com/conan-io/conan-center-index/pull/24894/commits/00457e341a1bc4c1be52db39cdb8e091a5461737).